autograder/api/courses/users/enroll.py

```diff
@@ -0,0 +1,41 @@
+import autograder.api.common
+import autograder.api.config
+
+API_ENDPOINT = 'courses/users/enroll'
+API_PARAMS = [
+    autograder.api.config.PARAM_USER_EMAIL,
+    autograder.api.config.PARAM_USER_PASS,
+    autograder.api.config.PARAM_COURSE_ID,
+
+    autograder.api.config.PARAM_DRY_RUN,
+
+    autograder.api.config.APIParam('skip-inserts',
+        'Skip inserts (default: False).',
+        required = False,
+        parser_options = {'action': 'store_true', 'default': False}),
+
+    autograder.api.config.APIParam('skip-updates',
+        'Skip updates (default: False).',
+        required = False,
+        parser_options = {'action': 'store_true', 'default': False}),
+
+    autograder.api.config.APIParam('send-emails',
+        'Send any emails.',
+        required = True, cli_param = False),
```

Comment:
I think these can be shared with users/upsert
(so these should probably be moved to autograder/api/config.py.